gateway/README.md

```diff
@@ -263,7 +263,7 @@ For this, you will need to specify which port to bind to.  An example config:
 ##### Add Tags to Spans
 
 The SignalFx listener has the ability to add tags to every span that passes through it.
-`AdditionalSpanTags` defines a set of tag name/value pairs that will be included on every span.
+`AdditionalSpanTags` defines a set of tag name/value pairs that will be included on every span. For more information, see :ref:`Advanced Smart Gateway Features <apm-advanced-gateway-features>`.
```

Comment:
You can't use rst format (:ref:) in MD files. Have to use the md link format (you can see it elsewhere in this doc) and link to the docs file directly - https://docs....